### PR TITLE
Info about shrinking

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Info about shrinking

## Type of change

- Documentation update

## Testing steps

N/A

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

**Review by juandspy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -39,17 +39,22 @@ CCX Notification Writer service
...
+into a database table named `new_reports`. Multiple reports can be consumed and
 written into the database for the same cluster, because the primary (compound)
 key for `new_reports` table is set to the combination `(org_id, cluster,
 updated_at)`. When some message does not conform to expected schema (for
-example if `org_id` is missing), such message is dropped and the error is
-stored into log.
+example if `org_id` is missing for any reason), such message is dropped and the
+error message with all relevant information about the issue is stored into the
+log. Messages are expected to contain `report` body represented as JSON.
+This body is shrink before it's stored into database so the database
```
Comment (epapbak) [inline README.md:49]: shrunk

File: docs/index.md [hunk 2]
```diff
@@ -8,12 +8,15 @@ nav_order: 0
...
+into a database table named `new_reports`. Multiple reports can be consumed and
 written into the database for the same cluster, because the primary (compound)
 key for `new_reports` table is set to the combination `(org_id, cluster,
 updated_at)`. When some message does not conform to expected schema (for
-example if `org_id` is missing), such message is dropped and the error is
-stored into log.
+example if `org_id` is missing for any reason), such message is dropped and the
+error message with all relevant information about the issue is stored into the
+log. Messages are expected to contain `report` body represented as JSON.
+This body is shrink before it's stored into database so the database
```
Comment (epapbak) [inline docs/index.md:18]: shrunk

**Review by epapbak — APPROVED** [review]
small tense comment

File: README.md (hunk 1 above)
Comment (tisnik) [reply README.md]: yeah my Czenglish ;) fixed and thank you!

File: docs/index.md (hunk 2 above)
Comment (tisnik) [reply docs/index.md]: fixed

**Pull request merged**